package/scripts/watch.ts

```diff
@@ -1,33 +1,43 @@
 import path from 'node:path';
 import * as fs from 'node:fs/promises';
+import { mkdirSync } from 'node:fs';
 
 import concurrently from 'concurrently';
 
-import { copyRunTimeDepsToOut, packageFolder } from './lib';
+import { copyLanguageServerFiles, packageFolder } from './lib';
 
 async function main() {
     console.log('Copying static dependencies...');
+
+    // Create this folder first to avoid a race condition in it's creation by
+    // the following fs.cp() calls
+    mkdirSync(path.join(packageFolder, 'out/vs'), { recursive: true });
+
     await Promise.all([
         fs.cp(path.dirname(require.resolve('monaco-editor/dev/vs/loader.js')), path.join(packageFolder, './out/vs'), {
             recursive: true,
+            filter(source) {
+                return !source.includes('/basic-languages') && !source.includes('/language');
+            },
         }),
 
-        copyRunTimeDepsToOut('out/vs/language/kusto'),
+        copyLanguageServerFiles('out/vs/language/kusto'),
 
         // This is super weird. Why do we do this?
         fs.writeFile(path.join(packageFolder, 'test/mode.txt'), 'dev'),
     ]);
 
-    // This is super weird. Why do we do this?
-    fs.writeFile(path.join(packageFolder, 'test/mode.txt'), 'dev'),
-        concurrently(
-            [
-                'live-server ./',
-                'tsc -w -p ./scripts/tsconfig.watch.json',
-                'yarn rollup -c ./scripts/rollup.dev.js -w --bundleConfigAsCjs',
-            ],
-            { cwd: packageFolder }
-        );
+    // This creates a race condition between live-sever and rollup 😢. App
+    // starts broken and contributors need to wait for rollup and then refresh
+    // the page
```

Comment:
I was trying to see how vitejs was doing this because they use rollup and serve up their assets but I couldn't figure it out. They have a hand rolled dev server which I don't think is worth it here.

Reply:
Their likely calling the rollup api directly rather than the cli.

This is absolutely fixable, but it would be a much larger change.

Reply:
Yep, the additional work is not worth it. We'll just have to live with it for now.